src/BenchmarkDotNet/Jobs/JobExtensions.cs

```diff
@@ -179,6 +179,13 @@ public static class JobExtensions
         /// </summary>
         public static Job WithPowerPlan(this Job job, PowerPlan powerPlan) => job.WithCore(j => j.Environment.PowerPlanMode.PowerPlan = powerPlan);
 
+        /// <summary>
+        /// Setting power plans by guid.
+        /// The default value is HighPerformance.
+        /// <remarks>Only available for Windows.</remarks>
+        /// </summary>
+        public static Job WithPowerPlan(this Job job, string powerPlanGuid) => job.WithCore(j => j.Environment.PowerPlanMode.PowerPlanGuid = powerPlanGuid);
```

Comment:
IMHO it would be better to pass a `Guid` here, not a `string` (then there will be no place for mistake)

Reply:
> 
> 
> IMHO it would be better to pass a `Guid` here, not a `string` (then there will be no place for mistake)

Agree, thx. I will fix it.